src/search/FileFilters.js

```diff
@@ -490,12 +491,13 @@ define(function (require, exports, module) {
         _picker.on("listRendered", _handleListRendered);
         
         _picker.on("select", function (event, item, itemIndex) {
+            // Trigger a exclusionSetChange Event which will be handled by findBar.
+            exports.trigger("exclusionSetChange");
```

Comment:
It seems like a cleaner approach to have the specific object returned by `createFilterPicker()` dispatch this event, since it's really specific to one instance of the picker UI.

OTOH, because of prefs in a certain sense it's _not_ instance-specific at all.  So an alternative would be to remove this code and just have FindInFilesUI listen for changes to the `"activeFileFilter"` view-state preference.